index.html

```diff
@@ -6,21 +6,32 @@
         <link rel="stylesheet" href="css/style.css">
```

Comment:
Here is my banner for winning:

    <div ng-show="win" ng-class="{'has-success': win}">
        Congrats! You Won!
    </div>